.github/workflows/ci.yml

```diff
@@ -9,9 +9,9 @@ name: Continuous Integration
 
 on:
   pull_request:
-    branches: ['**']
+    branches: ['**', '!update/**']
```

Comment:
This took me aback at first, because I thought it meant it would not run CI for PRs from branches named `update/**`. But, no, it won't run CI for PRs targeting branches named `update/**`.

Reply:
Thanks for clarifying this. Can't imagine how many times people were confused with this. Maybe it's just an attempt to get more traffic to docs? 😆